apps/chat/src/components/Folder/Folder.tsx

```diff
@@ -228,6 +228,15 @@ const Folder = <T extends ConversationInfo | PromptInfo | DialFile>({
     return filteredChildFolders.length > 0 || filteredChildItems.length > 0;
   }, [filteredChildFolders.length, filteredChildItems.length]);
 
+  const hasChildItemOnAnyLevel = useMemo(() => {
+    const prefix = `${currentFolder.id}/`;
+    return allItemsWithoutFilters.some(
+      (entity) =>
+        entity.folderId === currentFolder.id ||
+        entity.folderId.startsWith(prefix),
```

Comment:
Why folderId check is not enough?

Reply:
@mikitabut , because we need to check if child entity exists on any levels